spec/requests/api/requests_spec.rb

```diff
@@ -266,11 +246,17 @@
       request.add_tag(t.name, t.children.first.name)
 
       api_basic_authorize action_identifier(:requests, :read, :resource_actions, :get)
-      run_get requests_url(request.id), :attributes => "v_allowed_tags"
+      run_get requests_url(request.id), :attributes => "workflow,v_allowed_tags,v_class"
 
-      expected = a_hash_including("id" => request.id, "v_allowed_tags" => [a_hash_including("children")])
+      expected_workflow = a_hash_including("id" => request.id, "workflow" => a_hash_including("values"))
+      expected_tags = a_hash_including("id" => request.id, "v_allowed_tags" => [a_hash_including("children")])
+      expected_class = a_hash_including(
+        "id"      => request.id,
+        "v_class" => {"instance_logger" => a_hash_including("klass" => request.workflow.class.to_s)})
 
-      expect(response.parsed_body).to match(expected)
+      expect(response.parsed_body).to match(expected_workflow)
+      expect(response.parsed_body).to match(expected_tags)
+      expect(response.parsed_body).to match(expected_class)
```

Comment:
Might be better to have a single expect() for all the above since we're retesting id anyways, might something like:

```
expected_response = a_hash_including(....)

expect(response.parsed_body).to match(expected_response)
```